master/determined.code-workspace

```diff
@@ -1,13 +1,23 @@
 {
-	"folders": [
-		{
-			"path": "."
-		},
-		{
-			"path": "../agent"
-		},
-		{
-			"path": "../proto"
-		}
-	]
+  "folders": [
```

Comment:
this definitely was accidentally added in the checkpoints work, but if we wanna keep it around as a shared one, cool. (it was my pr, but it is not my workspace so idk how it got there, a few people worked on the pr)

Reply:
Yeah will remove